src/core/fonts.js

```diff
@@ -2903,6 +2903,7 @@ class Font {
       unicode = String.fromCharCode(unicode);
     }
 
+    const isSpace = unicode === " ";
```

Comment:
Given the rendering regressions, which I'm pretty certain are caused by the `isSpace` change, a naive idea would be to revert this and instead add a new `isUnicodeSpace` property to `Glyph`-instances for the `unicode === " ";`-case necessary for the text-selection logic.

Reply:
I don't know how I missed the failing tests... likely I was too focused on spaces.